.github/workflows/release.yml

```
@@ -0,0 +1,74 @@
+        run: mvn git-changelog-maven-plugin:git-changelog
+      - name: Commit Changelog
+        run: |
+          git add CHANGELOG.md
+          git commit -m "docs: update changelog for version ${{ github.event.inputs.releaseVersion }}"
+          git push origin main
+      - name: Configure Git user
+        run: |
+          git config user.email "actions@github.com"
+          git config user.name "GitHub Actions"
+      - name: Prepare Release
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1083:warning:4:19: This { is literal. Check expression (missing ;/\n?) or quote it

(shellcheck)